contracts/airdrop.sol

```diff
@@ -64,17 +50,10 @@ contract Airdrop is FraudProofHelpers {
             bool
         )
     {
-        require(
-            _txBytes.length == signatures.length,
-            "Mismatch length of signature and txs"
+        uint256 length = txs.transfer_size();
```

Comment:
Headsup here: We use the transfer in airdrop here. This affects internal working but does not change how the client consumes the API.